pkgs/development/python-modules/pyclip/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, pytest
+, xclip
+, xvfb-run
+}:
+
+buildPythonPackage rec {
+  pname = "pyclip";
+  version = "0.5.4";
+
+  src = fetchFromGitHub {
+    owner = "spyoungtech";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "19ff9cgnfx03mbmy5zpbdi986ppx38a5jf97vkqnic4g5sd1qyrn";
+  };
+
+  preConfigure = ''
+    sed -i 's@docs/README.md@README.md@g' setup.py
+  '';
+
+  checkInputs = [ pytest xclip xvfb-run ];
+
+  checkPhase = ''
+    xvfb-run -s '-screen 0 800x600x24' pytest tests
+  '';
+
+  meta = with lib; {
+    description = "Cross-platform clipboard utilities supporting both binary and text data.";
```

```suggestion
    description = "Cross-platform clipboard utilities supporting both binary and text data";
```

see `meta.description` part of https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md